### PR TITLE
Add Small Full Steps to Complete Controller

### PR DESCRIPTION
**Pull request by BSaunders95**
Required for SFA-123.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import uk.gov.companieshouse.web.accounts.util.RequestPathResolver;
+
+@Controller
+@RequestMapping(value = "/company/{companyNumber}/small-full/steps-to-complete")
+public class StepsToCompleteController {
+
+    private static final String TEMPLATE = "smallfull/stepsToComplete";
+
+    @Autowired
+    TransactionService transactionService;
```
Comment (jsimons1989) [inline src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java]: Can you set the private modifier on the private variables please

File: src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteControllerTests.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class StepsToCompleteControllerTests {
+
+    private MockMvc mockMvc;
+
+    @Mock
+    TransactionService transactionService;
+
+    @InjectMocks
+    StepsToCompleteController controller;
```
Comment (jsimons1989) [inline src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteControllerTests.java]: Same here (private modifier) please

File: src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+        return TEMPLATE;
+    }
+
+    @PostMapping()
+    public String postStepsToComplete(@PathVariable String companyNumber) {
+
+        Transaction transaction = transactionService.createTransaction(companyNumber);
+
+        // TODO: perform a redirect when a navigation engine is implemented
```
Comment (jsimons1989) [inline src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java]: Sorry to be picky: can you remove the "engine" from the wording of the TODO please? I would prefer to keep it simple and avoid needing an engine.

File: src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java (hunk 1 above)
Comment (BSaunders95) [reply src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java]: Done

File: src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteControllerTests.java (hunk 2 above)
Comment (BSaunders95) [reply src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteControllerTests.java]: Done

File: src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java (hunk 3 above)
Comment (BSaunders95) [reply src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/StepsToCompleteController.java]: Done

**Pull request merged**